packages/odyssey-react/src/components/Infobox/Infobox.tsx

```diff
@@ -22,7 +23,7 @@ import styles from "./Infobox.module.scss";
 interface CommonProps
   extends Omit<
     ComponentPropsWithRef<"aside">,
-    "style" | "className" | "children" | "title" | "content"
+    "style" | "className" | "children" | "title" | "content" | "color"
```

Comment:
We should build a simple generic type that allows this type of repeated type definition to be defined in one place. DRY FTW

Going further, we should also add "color" to the omit list for `omitProps` so the type annotations and run time behavior match

Also consider that props that are available on `aside`, like `role` are not available here (as we hardcode it below). That is one more place we can improve this change

Reply:
There's a subtlety in play here with regard to omit, color needs to be omitted because it's [a valid (but legacy) attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes) on html elements that has [a different definition than ours](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1872).

There may be cases where we don't want color omitted from passed props, but we have to omit it from what's being inherited from the html element.